clusterloader2/testing/huge-service/modules/measurements.yaml

```diff
@@ -70,6 +78,19 @@ steps:
       defaultAllowedRestarts: {{$ALLOWED_CONTAINER_RESTARTS}}
       customAllowedRestarts: {{YamlQuote $CUSTOM_ALLOWED_CONTAINER_RESTARTS 4}}
 {{end}}
+{{if $ENABLE_DNS_PROPAGATION_MEASUREMENT}}
+  - Identifier: DnsPropagation
+    Method: DnsPropagation
+    Params:
+      action: {{$action}}
+      DNSPropagationProbeStatefulSet: {{.serviceName}}
```

Comment:
Nit: Please create variables at the top (like with $statefulsetEndpoints)

Reply:
Fixed, thanks!